docs/mining/overview.md

```diff
@@ -26,74 +26,141 @@ transactions waiting to be processed in the mempool.
 
-* :octicons-mark-github-16: [cgminer](https://github.com/kR105-zz/cgminer) - cgminer is a popular miner for **AMD** GPUs that has a long history of use in many different cryptocurrencies. It is more difficult to use than the decred gominer.
 
-* :octicons-mark-github-16: [ccminer](https://github.com/tpruvot/ccminer) - ccminer is a popular miner for **NVIDIA** GPUs that has a long history of use in many different cryptocurrencies. It is more difficult to use than the decred gominer.
+## Choosing and configuring a mining pool 
 
-* :octicons-mark-github-16: [sgminer](https://github.com/tpruvot/sgminer) - Those with an **AMD** graphics card running Windows may want to download sgminer.
+To finish the configuration, you will need to choose a mining pool. You can consider payout method ([PPS vs PPLNS](https://www.hashrateindex.com/blog/pps-fpps-pplns-pps_plus)), pool fee, geographic location of the pool, ping time to the server, user interface, statistics / data, customer support, community channels and more. Mining pools all work more or less the same but you may wish to sign up at multiple pools and see which one suits you the best.
 
----
+These mining pools are known to support Decred:
+
+
+
+*   **[Luxor](https://mining.luxor.tech/coins/decred)** (USA)
+*   **[F2Pool](https://www.f2pool.com/)** (China)
+*   **[Poolin](https://www.poolin.com/)** (China)
+*   **[UUpool](https://uupool.cn/dcr)** (China)
+*   **[Antpool](https://antpool.com/)** (China)
+*   **[BTC.com](https://btc.com/)** (China)
+
+In this guide, we will set up an ASIC with [Luxor](https://mining.luxor.tech/), a US-based mining pool that operates based on the pay-per-share (PPS) payment method. 
+
+**Completing Configuration**
+
+Once you have created a [user account](https://mining.luxor.tech/account/signup) with the mining pool, you will have all the information you need to complete the setup process.
+
+*   URL: stratum+tcp://dcr.luxor.tech:700
+*   Worker: [LuxorUsername].WorkerName Workername can be anything, but avoid using symbols or special characters as it may be invalid.
+*   Password: 123
+
+Once you have filled out the details, click Save & Apply. Setup is now complete.
+
+**Monitoring Results**
+
+It will take about 5 minutes for your workers to appear on the stats page. To find your user, simply go to Luxor, login to your account and navigate to the Workers tab. You should see something like this:
+
+
+![Monitoring mining results with Luxor pool](../img/pow-mining-guide/monitoring-dashboard.png?raw=true)
 
-## <img class="dcr-icon" src="/img/dcr-icons/Dcrtl.svg" /> Running the Software
+## Obtain a Decred Address to Receive Funds 
```

Comment:
We can remove this section and the link to wallet setup.

Reply:
Removed